content/en/docs/guides/metadata.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Metadata
+description: >-
+  Explains what metadata is, how it is transmitted, and what it is used for.
+---
+
+### Overview
+
+Metadata is a side channel that allows clients and servers to provide
+information to each other that is associated with the data messages.
```

Comment:
"associated with the data messages" could be misleading, making someone think it is available for each message. The "sent with a gRPC request or response" would reinforce that view.

Reply:
reworded